src/managers/slidedownManager/SlidedownManager.ts

```diff
@@ -51,13 +51,17 @@ export class SlidedownManager {
       isSlidedownPushDependent = PromptsHelper.isSlidedownPushDependent(slidedownType);
     }
 
-    // applies to push slidedown type only
-    if (slidedownType === DelayedPromptType.Push && isSubscribed) {
-      return false;
-    }
-
     // applies to both push and category slidedown types
     if (isSlidedownPushDependent) {
+      if (isSubscribed) {
```

Comment:
This change largely fixes the problem. In a future PR we probably want to use the browser's Permission state instead but it might not be easy to do here.